templates/zerver/help/delete-a-message.md

```diff
@@ -0,0 +1,7 @@
+# Delete a message
+
+{!admin-only.md!}
+{!message-actions.md!}
+{!down-chevron.md!}
+
+2. Select the **Delete message** option from the dropdown to delete that message.
```

Comment:
the text is very clear, though it does cover both admin- and user- directed deletion, which has its own help page at `Edit or delete a message`. i was thinking of keeping those two separate in order to not make the help page too long, and maybe referencing each other-- does that make sense?